iree/compiler/Dialect/Flow/Transforms/DispatchConfig.h

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "iree/compiler/Dialect/Flow/Analysis/Dispatchability.h"
+#include "mlir/IR/Operation.h"
+
+namespace mlir {
+namespace iree_compiler {
+namespace IREE {
+namespace Flow {
+
+// Queries dispatch options for an operation.
+// This is presently mostly a hard-coded set of heuristics but should expand
+// to be based on both queries of new op interfaces and a cost model.
+class OpDispatchPolicy {
+ public:
+  using AnchorBenefit = int;
```

Comment:
add comment on whether 0 = highest or lowest (or whatever the scale may be, which I feel like for these things always ends up as a cold war-style one-upsmanship game :)